podium-gateway/src/main/webapp/app/request/form/request-form.route.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2017. The Hyve and respective contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * See the file LICENSE in the root of this repository.
+ *
+ */
+
+import { Route } from '@angular/router';
+
+import { UserRouteAccessService } from '../../shared';
+import { RequestFormComponent } from './request-form.component';
+
+export const requestFormRoute: Route = {
+  path: 'new',
```

Comment:
maybe more descriptive something like "new-request" or "create-request"? unless if this is sub path like /request/new